debian/ooniprobe-cli.service

```diff
@@ -0,0 +1,51 @@
+[Unit]
+Description=OONI CLI Probe
+Documentation=man:ooniprobe-cli
+#Documentation=file:///usr/share/doc/ooniprobe-cli/html/index.html
+Documentation=https://ooni.org/
+After=network.target tor.target
+Wants=network-online.target
+ConditionPathExists=/etc/ooniprobe/ooniprobe.conf
+
+[Service]
+Type=simple
+ExecStart=/usr/bin/ooniprobe --config=/etc/ooniprobe/ooniprobe.conf run all
+TimeoutStopSec=5
+KillMode=mixed
+
+Environment="OONI_HOME=/var/lib/ooniprobe"
+User=ooniprobe
+PermissionsStartOnly=true
+Restart=on-abnormal
+RestartSec=2s
+LimitNOFILE=65536
+WorkingDirectory=/var/lib/ooniprobe
+
+# Sandboxing
+CapabilityBoundingSet=CAP_NET_BIND_SERVICE
+LockPersonality=yes
+NoNewPrivileges=yes
+PrivateDevices=yes
+PrivateTmp=yes
+PrivateUsers=yes
+ProtectClock=yes
+ProtectControlGroups=yes
+ProtectHome=yes
+ProtectHostname=yes
+ProtectKernelLogs=yes
+ProtectKernelModules=yes
+ProtectKernelTunables=yes
+ProtectSystem=full
+ReadOnlyDirectories=/
+ReadWriteDirectories=-/proc
+ReadWriteDirectories=-/var/log/ooniprobe
+ReadWriteDirectories=-/var/lib/ooniprobe
+ReadWriteDirectories=-/var/run
```

Comment:
My systemd-fu is such that I don't understand the `-` here. The [manual to which I have access](https://www.commandlinux.com/man-page/man5/systemd.exec.5.html) reads:

> Paths in ReadOnlyDirectories= and InaccessibleDirectories= may be prefixed with "-", in which case they will be ignored when they do not exist. Note that using this setting will disconnect propagation of mounts from the service to the host (propagation in the opposite direction continues to work). This means that this setting may not be used for services which shall be able to install mount points in the main mount namespace.

I'm clearly missing some knowledge here, please help :-).

Reply:
If those directories are missing for any reason, the systemd unit file will not break. E.g. /var/log/ooniprobe is not being created and that's ok.

Reply:
What confuses me is that the manual page that I linked to explicitly states that this behaviour is only occurring for ReadOnlyDirectories and InaccessibleDirectories, but above it's used for ReadWriteDirectories. It seems the [Debian manual page for that topic](https://manpages.debian.org/stretch/systemd/systemd.exec.5.en.html) says that also ReadWriteDirectories can be used along with the `-` modifier.